src/components/Users/ManageUsers.tsx

```diff
@@ -178,7 +178,54 @@ export default function ManageUsers() {
       </div>
 
       <div className="pt-4">
-        <div>{manageUsers}</div>
+        <div className="mb-4 flex flex-col items-center justify-between gap-3 sm:flex-row">
+          <div className="sm:w-1/2">
+            <SearchByMultipleFields
+              id="search-by-username"
+              className="w-full"
+              onSearch={(key, value) => updateQuery({ username: value })}
+              options={[
+                {
+                  key: "username",
+                  label: "username",
+                  type: "text" as const,
+                  placeholder: t("search_by_username"),
+                  value: qParams.username || "",
+                  shortcutKey: "u",
+                },
+              ]}
+              clearSearch={clearSearch}
+            />
+          </div>
+          <Tabs
```

Comment:
Let's switch to using Shadcn's Tabs while we are at it.

Reply:
You can install it, it's being added `form-field-v1` if you need reference.

Reply:
Hey @Jacobjeevan, everything is set! Just wondering if I should ping to get the default design for Shadcn tabs? Here’s the image for reference 👇
<img width="176" alt="Screenshot 2024-12-24 at 12 10 23 PM" src="https://github.com/user-attachments/assets/355e91a7-f229-4700-a260-1fea7aae3d71" />

Reply:
PS: please remove `merge conflict` label 😅

Reply:
> Hey @Jacobjeevan, everything is set! Just wondering if I can ping to get the default design for Shadcn tabs? Here’s the image for reference 👇 <img alt="Screenshot 2024-12-24 at 12 10 23 PM" width="176" src="https://private-user-images.githubusercontent.com/148325158/398342457-355e91a7-f229-4700-a260-1fea7aae3d71.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwMzQxMDcsIm5iZiI6MTczNTAzMzgwNywicGF0aCI6Ii8xNDgzMjUxNTgvMzk4MzQyNDU3LTM1NWU5MWE3LWYyMjktNDcwMC1hMjYwLTFmZWE3YWFlM2Q3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQwOTUwMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTdmNjllYjEzYjYxMDE5OGY5ZmYwZDQyYTdjMGZhNDlmMWMxYzY4MzIxOGQxZTk4OWI5OTM5MGZlMmYzMjMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-yGiUM5LoxKV8cUKOmLv3fP1BZIVnC5xFhzFaKFKOL8">

Current design is fine 👍  It's in line with new changes.